Tests/SonarQube.MSBuild.Tasks.IntegrationTests/E2ETests/E2EAnalysisTests.cs

```diff
@@ -703,17 +702,6 @@ private void CheckAnalysisFileList(ProjectDescriptor expected, string projectOut
             }
         }
 
-        private static void AssertFileIsNotAnalysed(string analysisFileListPath, string unanalysedPath)
-        {
-            var actualFiles = GetAnalysedFiles(analysisFileListPath); CollectionAssert.DoesNotContain(actualFiles, unanalysedPath, "File should not be analysed: {0}", unanalysedPath);
-        }
-
-        private static void AssertFileIsAnalysed(string analysisFileListPath, string unanalysedPath)
-        {
-            var actualFiles = GetAnalysedFiles(analysisFileListPath);
-            CollectionAssert.Contains(actualFiles, unanalysedPath, "File should not be analysed: {0}", unanalysedPath);
-        }
-
         private static string[] GetAnalysedFiles(string analysisFileListPath)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused private method 'GetAnalysedFiles'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1144)